ansible_collections/arista/cvp/docs/how-to/cvp-authentication.md

```diff
@@ -28,6 +28,17 @@ ansible_httpapi_use_ssl: true
 ansible_httpapi_validate_certs: false
```

Comment:
line 16 could/should be changed to `ansible_password` as well.

Reply:
sure I'll change that too
re api_token, it was introduced in 1.0.5 as per https://github.com/aristanetworks/cvprac/commit/409b68d905850bd471b0355b2574cf4497579ada so we won't need to change the requirements